recipe/meta.yaml

```diff
@@ -27,9 +28,6 @@ test:
     - {{ compiler('fortran') }}
   files:
     - test/fftw_dft_r2c_1d_c_example.c
-    - test.sh
-  commands:
-    - chmod u+x test.sh && ./test.sh #[unix]
 
 about:
   home: https://developer.arm.com/tools-and-software/server-and-hpc/downloads/arm-performance-libraries
```

Comment:
There are many names of the applicable
licenses under the hood https://developer.arm.com/tools-and-software/server-and-hpc/downloads/arm-performance-libraries/eula. I think all of them must be in the recipe and the EULA as per the SPDX format